traceloader.py

```diff
@@ -116,22 +128,26 @@ def _link_trace_multibw_generator(self):
             dict: information of traces
         '''
         tools.clear_folder(self._trace_folder)
-        link_trace, trace_info = self._generate_mmlink_multibw(self._trace_list)
-        for trace in link_trace:
-            with open(os.path.join(self._trace_folder, trace), 'w', encoding='utf-8') as f:
-                f.write(link_trace[trace])
+        link_trace, trace_info = generate_mmlink_multibw(self._trace_list)
```

Comment:
You can assign self.link_trace of self.trace_info here and change the corresponding codes matter, no need to wait until the end of the function. Remove unnecessary variables.